docs/book/src/developer/providers/v1.0-to-v1.1.md

```diff
@@ -29,7 +31,9 @@ are kept in sync with the versions used by `sigs.k8s.io/controller-runtime`.
 
 * Some controllers have been moved to internal to reduce there API surface. We now only
```

Comment:
Not from this PR, but I'll mention just in case there is some other need for an update and you feel like addressing this, but "there API" should be "their API".

Reply:
Thx, always! Big fan of the boy scout rule and fixing typos or errors like this :0